// @esri/calcite-components/dist/cjs/id-a6d35d75.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v2.7.1
 */
'use strict';

const calciteInputTimePicker_calciteTimePicker_entry = require('./calcite-input-time-picker.calcite-time-picker-87e98896.js');
require('./index-6469e98a.js');
require('./form-dc8eab44.js');
require('./dom-487ef37d.js');
require('./guid-0cfff064.js');
require('./resources-661dfdf1.js');
require('./interactive-a2b73062.js');
require('./browser-bd0cc8a3.js');
require('./key-92fad604.js');
require('./label-75f13c02.js');
require('./component-2d355ea4.js');
require('./loadable-58c11853.js');
require('./locale-82485763.js');
require('./observers-54af6eba.js');
require('./focusTrapComponent-b70a59e2.js');
require('./math-dd0998db.js');
require('./t9n-42cef130.js');
require('./openCloseComponent-944b93cd.js');
require('./Validation-80b35f71.js');

// Indonesian [id]
var locale = {
  name: 'id',
  weekdays: 'Minggu_Senin_Selasa_Rabu_Kamis_Jumat_Sabtu'.split('_'),
  months: 'Januari_Februari_Maret_April_Mei_Juni_Juli_Agustus_September_Oktober_November_Desember'.split('_'),
  weekdaysShort: 'Min_Sen_Sel_Rab_Kam_Jum_Sab'.split('_'),
  monthsShort: 'Jan_Feb_Mar_Apr_Mei_Jun_Jul_Agt_Sep_Okt_Nov_Des'.split('_'),
  weekdaysMin: 'Mg_Sn_Sl_Rb_Km_Jm_Sb'.split('_'),
  weekStart: 1,
  formats: {
    LT: 'HH.mm',
    LTS: 'HH.mm.ss',
    L: 'DD/MM/YYYY',
    LL: 'D MMMM YYYY',
    LLL: 'D MMMM YYYY [pukul] HH.mm',
    LLLL: 'dddd, D MMMM YYYY [pukul] HH.mm'
  },
  relativeTime: {
    future: 'dalam %s',
    past: '%s yang lalu',
    s: 'beberapa detik',
    m: 'semenit',
    mm: '%d menit',
    h: 'sejam',
    hh: '%d jam',
    d: 'sehari',
    dd: '%d hari',
    M: 'sebulan',
    MM: '%d bulan',
    y: 'setahun',
    yy: '%d tahun'
  },
  ordinal: function ordinal(n) {
    return n + ".";
  }
};
calciteInputTimePicker_calciteTimePicker_entry.dayjs.locale(locale, null, true);

exports.default = locale;
