// @esri/calcite-components/dist/esm/bs-004af180.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v2.7.1
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-ab6b9cb6.js';
import './index-28d8549d.js';
import './form-034629b4.js';
import './dom-bef18e4a.js';
import './guid-617f6e1b.js';
import './resources-25959fe9.js';
import './interactive-583e13d7.js';
import './browser-1baf23f7.js';
import './key-3f1e3da4.js';
import './label-8c0f9304.js';
import './component-b0a9a44a.js';
import './loadable-6056dcbc.js';
import './locale-b835bf72.js';
import './observers-ff65e337.js';
import './focusTrapComponent-623f3c57.js';
import './math-a5520ff4.js';
import './t9n-91f18f76.js';
import './openCloseComponent-f04579aa.js';
import './Validation-cf2abfd2.js';

// Bosnian [bs]
var locale = {
  name: 'bs',
  weekdays: 'nedjelja_ponedjeljak_utorak_srijeda_četvrtak_petak_subota'.split('_'),
  months: 'januar_februar_mart_april_maj_juni_juli_august_septembar_oktobar_novembar_decembar'.split('_'),
  weekStart: 1,
  weekdaysShort: 'ned._pon._uto._sri._čet._pet._sub.'.split('_'),
  monthsShort: 'jan._feb._mar._apr._maj._jun._jul._aug._sep._okt._nov._dec.'.split('_'),
  weekdaysMin: 'ne_po_ut_sr_če_pe_su'.split('_'),
  ordinal: function ordinal(n) {
    return n;
  },
  formats: {
    LT: 'H:mm',
    LTS: 'H:mm:ss',
    L: 'DD.MM.YYYY',
    LL: 'D. MMMM YYYY',
    LLL: 'D. MMMM YYYY H:mm',
    LLLL: 'dddd, D. MMMM YYYY H:mm'
  }
};
dayjs.locale(locale, null, true);

export default locale;
