// @esri/calcite-components/dist/esm/da-c776ab95.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v2.7.1
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-ab6b9cb6.js';
import './index-28d8549d.js';
import './form-034629b4.js';
import './dom-bef18e4a.js';
import './guid-617f6e1b.js';
import './resources-25959fe9.js';
import './interactive-583e13d7.js';
import './browser-1baf23f7.js';
import './key-3f1e3da4.js';
import './label-8c0f9304.js';
import './component-b0a9a44a.js';
import './loadable-6056dcbc.js';
import './locale-b835bf72.js';
import './observers-ff65e337.js';
import './focusTrapComponent-623f3c57.js';
import './math-a5520ff4.js';
import './t9n-91f18f76.js';
import './openCloseComponent-f04579aa.js';
import './Validation-cf2abfd2.js';

// Danish [da]
var locale = {
  name: 'da',
  weekdays: 'søndag_mandag_tirsdag_onsdag_torsdag_fredag_lørdag'.split('_'),
  weekdaysShort: 'søn._man._tirs._ons._tors._fre._lør.'.split('_'),
  weekdaysMin: 'sø._ma._ti._on._to._fr._lø.'.split('_'),
  months: 'januar_februar_marts_april_maj_juni_juli_august_september_oktober_november_december'.split('_'),
  monthsShort: 'jan._feb._mar._apr._maj_juni_juli_aug._sept._okt._nov._dec.'.split('_'),
  weekStart: 1,
  ordinal: function ordinal(n) {
    return n + ".";
  },
  formats: {
    LT: 'HH:mm',
    LTS: 'HH:mm:ss',
    L: 'DD.MM.YYYY',
    LL: 'D. MMMM YYYY',
    LLL: 'D. MMMM YYYY HH:mm',
    LLLL: 'dddd [d.] D. MMMM YYYY [kl.] HH:mm'
  },
  relativeTime: {
    future: 'om %s',
    past: '%s siden',
    s: 'få sekunder',
    m: 'et minut',
    mm: '%d minutter',
    h: 'en time',
    hh: '%d timer',
    d: 'en dag',
    dd: '%d dage',
    M: 'en måned',
    MM: '%d måneder',
    y: 'et år',
    yy: '%d år'
  }
};
dayjs.locale(locale, null, true);

export default locale;
