// @esri/calcite-components/dist/esm/sv-274f6ae1.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v2.7.1
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-ab6b9cb6.js';
import './index-28d8549d.js';
import './form-034629b4.js';
import './dom-bef18e4a.js';
import './guid-617f6e1b.js';
import './resources-25959fe9.js';
import './interactive-583e13d7.js';
import './browser-1baf23f7.js';
import './key-3f1e3da4.js';
import './label-8c0f9304.js';
import './component-b0a9a44a.js';
import './loadable-6056dcbc.js';
import './locale-b835bf72.js';
import './observers-ff65e337.js';
import './focusTrapComponent-623f3c57.js';
import './math-a5520ff4.js';
import './t9n-91f18f76.js';
import './openCloseComponent-f04579aa.js';
import './Validation-cf2abfd2.js';

// Swedish [sv]
var locale = {
  name: 'sv',
  weekdays: 'söndag_måndag_tisdag_onsdag_torsdag_fredag_lördag'.split('_'),
  weekdaysShort: 'sön_mån_tis_ons_tor_fre_lör'.split('_'),
  weekdaysMin: 'sö_må_ti_on_to_fr_lö'.split('_'),
  months: 'januari_februari_mars_april_maj_juni_juli_augusti_september_oktober_november_december'.split('_'),
  monthsShort: 'jan_feb_mar_apr_maj_jun_jul_aug_sep_okt_nov_dec'.split('_'),
  weekStart: 1,
  yearStart: 4,
  ordinal: function ordinal(n) {
    var b = n % 10;
    var o = b === 1 || b === 2 ? 'a' : 'e';
    return "[" + n + o + "]";
  },
  formats: {
    LT: 'HH:mm',
    LTS: 'HH:mm:ss',
    L: 'YYYY-MM-DD',
    LL: 'D MMMM YYYY',
    LLL: 'D MMMM YYYY [kl.] HH:mm',
    LLLL: 'dddd D MMMM YYYY [kl.] HH:mm',
    lll: 'D MMM YYYY HH:mm',
    llll: 'ddd D MMM YYYY HH:mm'
  },
  relativeTime: {
    future: 'om %s',
    past: 'för %s sedan',
    s: 'några sekunder',
    m: 'en minut',
    mm: '%d minuter',
    h: 'en timme',
    hh: '%d timmar',
    d: 'en dag',
    dd: '%d dagar',
    M: 'en månad',
    MM: '%d månader',
    y: 'ett år',
    yy: '%d år'
  }
};
dayjs.locale(locale, null, true);

export default locale;
